tools/tasks/seed/build.tools.ts

```diff
@@ -0,0 +1,16 @@
+import * as util from 'gulp-util';
```

Comment:
The naming of this task is much better but let's keep the code from the previous one since it also includes `plugins.template`.